addons/payment/data/portal_entry_data.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+    <record id="portal_payment_orders" model="portal.entry">
+        <field name="name">Payment Methods</field>
+        <field name="description">Follow Payment</field>
```

Comment:
"Follow Payments" doesn't make sense for this card. Reuse the previous "Manage your payment methods" instead.